### PR TITLE
Fixes GH-1683

### LINKED ISSUE #1683: Docs: Special Type : any
```
let obj: any = { x: 0 };
// None of these lines of code are errors
obj.foo();
obj();
obj.bar = 100;
obj = "hello";
const n: number = obj;
```

In above example was mentioned that 'None of these lines of code are errors' but obj.foo() and obj() throw an error. 

Page url: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html

### PR DESCRIPTION
**Pull request by hahmadia**
Fixes GH-1683
Be more explicit about the error

### CODE REVIEW

File: packages/documentation/copy/en/handbook-v2/Everyday Types.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ When a value is of type `any`, you can access any properties of it (which will i
 
 ```ts twoslash
 let obj: any = { x: 0 };
-// None of these lines of code are errors
+// None of these lines of code are will throw compiler errors
+// That is none of these lines of code will have a red underline underneath them
+// However, this will still throw runtime errors.
```
Comment (orta) [inline packages/documentation/copy/en/handbook-v2/Everyday Types.md]:
```suggestion
// None of the following lines of code are will throw compiler errors.
// Using `any` disables all further type checking, and it is assumed 
// you know the environment better than TypeScript.
```

**Comment by orta** [comment]
Thanks! Did some minor copy tweaks but this is good to go

File: packages/documentation/copy/en/handbook-v2/Everyday Types.md [hunk 2]
```diff
@@ -42,7 +42,9 @@ When a value is of type `any`, you can access any properties of it (which will i
 
 ```ts twoslash
 let obj: any = { x: 0 };
-// None of these lines of code are errors
+// None of the following lines of code are will throw compiler errors.
```
Comment (hahmadia) [inline packages/documentation/copy/en/handbook-v2/Everyday Types.md]:
```suggestion
// None of the following lines of code will throw compiler errors.
```

**Comment by hahmadia** [comment]
@orta removed extra "are"

**Comment by orta** [comment]
Thanks

**Pull request merged**